include/oneapi/dpl/experimental/kt/esimd_radix_sort.h

```diff
@@ -31,6 +31,15 @@ template <bool _IsAscending, std::uint8_t _RadixBits, typename _KernelParam, typ
 sycl::event
 radix_sort(sycl::queue __q, _Range&& __rng, _KernelParam __param)
 {
+    static_assert(_RadixBits == 7 || _RadixBits == 8);
+
+    static_assert(param.data_per_workitem == 32 || param.data_per_workitem == 64 || param.data_per_workitem == 96 ||
+                  param.data_per_workitem == 128 || param.data_per_workitem == 160 || param.data_per_workitem == 192 ||
+                  param.data_per_workitem == 224 || param.data_per_workitem == 256 || param.data_per_workitem == 288 ||
+                  param.data_per_workitem == 320 || param.data_per_workitem == 352 || param.data_per_workitem == 384 ||
+                  param.data_per_workitem == 416 || param.data_per_workitem == 448 || param.data_per_workitem == 480 ||
+                  param.data_per_workitem == 512);
```

Comment:
Can you use `%` for simplicity?

Reply:
As for me better to have full list of available values without any calculations.